pyseeyou/locales.py

```diff
@@ -48,60 +80,830 @@ def get_parts_of_num(num_string):
     return n, i, v, w, f, t
 
 
-def brazilian_pt(n, i, v, w, f, t):
-    if 0 <= i < 2 and f == 0:
-        return 'one'
+# =================
+# CODE BY GENERATOR
+# =================
```

Comment:
With all this generated code, this file becomes quite long. I think it'd be better to split this out into its own file and then import that here.